includes/class-events.php

```diff
@@ -268,17 +268,29 @@ public function run_event( $timestamp, $action, $instance, $force = false ) {
 		$this->update_event_record( $event );
 
 		// Run the event
-		do_action_ref_array( $event->action, $event->args );
+		try {
+			do_action_ref_array( $event->action, $event->args );
+		} catch ( \Throwable $t ) {
+			$return = array(
+				'success' => false,
+				'message' => sprintf( __( 'Exception for job with action `%1$s` and arguments `%2$s` - %3$s in %4$s on line %5$d.', 'automattic-cron-control' ), $event->action, maybe_serialize( $event->args ), $t->getMessage(), $t->getFile(), $t->getLine() ),
```

Comment:
Small thought on the verbage here - the message says `Exception for job...`, when it's really a `Throwable`. My concern is that someone debugging might read that as 'the code threw an `Exception` and spend a while searching for thrown `Exceptions`, when it could have been a regular fatal.